.rubocop.yml

```diff
@@ -0,0 +1,78 @@
+AllCops:
+  TargetRubyVersion: 2.5.1
+  DisplayCopNames: true
+  Exclude:
+    - db/schema.rb
+    - db/migrate/**/*
+    - kithe.gemspec
+    - Rakefile
+    - Gemfile
+    - bin/rails
+    - spec/**/*
+    - test/**/*
+    - config/**/*
+    - lib/**/*
+    # - app/**/*
```

Comment:
let's include lib/ too

Reply:
Cool, added lib. Not sure what should and shouldn't be checked at this early stage and didn't want to flood the codebase with errors in generated code.